androidbrowserhelper/build.gradle

```diff
@@ -61,3 +62,41 @@ dependencies {
     androidTestImplementation 'com.linkedin.dexmaker:dexmaker-mockito:2.19.0'
     androidTestImplementation 'junit:junit:4.12'
 }
+
+publishing {
+    publications {
+        Production(MavenPublication) {
+            pom {
+                name = 'android-browser-helper'
+                url = 'https://github.com/GoogleChrome/android-browser-helper'
+                licenses {
+                    license {
+                        name = 'The Apache License, Version 2.0'
+                        url = 'http://www.apache.org/licenses/LICENSE-2.0.txt'
+                    }
+                }
+            }
+
+            artifact("$buildDir/outputs/aar/androidbrowserhelper-release.aar")
+
+            groupId 'com.google.androidbrowserhelper'
+            artifactId 'androidbrowserhelper'
+            version '0.1.0-alpha1'
+
+            pom.withXml {
+                def dependenciesNode = asNode().appendNode('dependencies')
+
+                // Iterate over the implementation dependencies (we don't want the test ones), adding a <dependency> node for each
```

Comment:
nit: line wrapping.

Reply:
Wrapped the line, but added a link that is large :(